src/trackinfoobject.cpp

```diff
@@ -783,13 +783,14 @@ bool TrackInfoObject::isDirty() {
 
 int TrackInfoObject::getRating() const {
     QMutexLocker lock(&m_qMutex);
-    return m_iRating;
+    return m_metadata.getRating();
```

Comment:
I have other projects on my todo list. Currently I am working on the native Jack interface.

Such a 'sync rating to stars' option seam to work. I am only unsure how we have to deal with the email address. Do we need to expose it to the user? 
@uklotzde any idea?